app/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -39,7 +39,7 @@
 	<string name="delete_successful_message">成功刪除已選歌詞檔</string>
 	<string name="delete_timestamps_and_lyrics_message">刪除時間標記及歌詞</string>
 	<string name="delete_timestamps_only_message">只刪除時間標記</string>
-	<string name="deleting_message">正在刪除... 這可能需要一些時間，取決於檔案與讀取位置的距離</string>
+	<string name="deleting_message">正在刪除... \n這可能需要一些時間，取決於檔案與讀取位置的距離</string>
```

Comment:
I don't think the newline character is required here as the dots provide enough seperation